analyzer/src/main/kotlin/managers/utils/NodeSupport.kt

```diff
@@ -135,6 +135,23 @@ fun readProxySettingsFromNpmRc(npmRc: String): ProtocolProxyMap {
     return map
 }
 
+const val PUBLIC_NPM_REGISTRY = "https://registry.npmjs.org"
```

Comment:
This is now unused in this commit, and should only be introduced in the next commit. Maybe, this constant should also be moved to `Npm.kt` instead. And while at it, I'd also propose to name it `DEFAULT_NPM_REGISTRY` instead, as there might be other publicly accessible NPM registries.